src/ipsis/course-merge/CourseMergeOfferingCollectionEntity.js

```diff
@@ -10,11 +10,11 @@ export class CourseMergeOfferingCollectionEntity extends BaseCollectionEntity {
 		if (!this._entity) {
 			return;
 		}
-		return this._entity.entities;
+		return allEntities ?? this._entity.entities;
```

Comment:
Can we also call it `_allEntities` to better indicate that it shouldn't be used outside the class?